cmd/Makefile.am

```diff
@@ -543,7 +543,11 @@ snap_gdb_shim_snap_gdbserver_shim_LDFLAGS = -static
 ##
 
 systemdsystemgeneratordir = $(SYSTEMD_SYSTEM_GENERATOR_DIR)
+EXTRA_PROGRAMS = snapd-generator/snapd-generator
```

Comment:
This seems like something that should be in a separate patch, it's a fairly clean split (unstage Makefile.am, configure.ac and the one-line change in snapcraft.yaml)

Reply:
Sure. This one can be separate. I will revise my commit history.

Reply:
Done.